### PR TITLE
cred manifest example

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

**Comment by decentralgabe** [comment]
didn't add signing yet

cc: @dcrousso

File: example/manifest/manifest.go [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+	"github.com/goccy/go-json"
+	"github.com/google/uuid"
+)
+
+// getDIDKey will return a DID key and its private key
+func getDIDKey() (gocrypto.PrivateKey, *did.DIDKey, error) {
+	return did.GenerateDIDKey(crypto.Ed25519)
+}
+
+// Prepare a credential schema that will be issued to issue a credential from a successful Credential Manifest
```
Comment (nitro-neal) [inline example/manifest/manifest.go:32]: nit:
// Prepare a credential schema that will be used to issue a credential from a successful Credential Manifest

File: example/manifest/manifest.go [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+	if err := builder.SetClaimFormat(exchange.ClaimFormat{
+		JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+	}); err != nil {
+		return nil, err
+	}
+
+	return builder.Build()
+}
+
+// Prepare a credential which is required to fill out the credential manifest's application
```
Comment (nitro-neal) [inline example/manifest/manifest.go:184]: nit:

// Prepare a credential which is required to fill out the credential manifest's application's input descriptor's requirements

**Review by nitro-neal — APPROVED** [review]
Awesome this is great!

File: example/manifest/manifest.go (hunk 2 above)
Comment (decentralgabe) [reply example/manifest/manifest.go:184]: tyty

File: example/manifest/manifest.go (hunk 1 above)
Comment (decentralgabe) [reply example/manifest/manifest.go:32]: tyty

**Pull request merged**